pkg/nodebootstrap/assets/kubeconfig.yaml

```diff
@@ -11,7 +11,7 @@ contexts:
 - name: kubelet
   context:
     cluster: kubernetes
-    user: kubelet  
+    user: kubelet
```

Comment:
As there is a change here please run `make generate` and commit the change to assets.go

Reply:
This file has been delete in #184

Reply:
Great, so rebasing on master should make the build pass then.

Reply:
Good spot @errordeveloper